app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = [/\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i].freeze
+  validates :email, presence: true,
+    format: {with: VALID_EMAIL_REGEX}
+  validates :name, presence: true,
+    length: {
+      minimum: Settings.user.namee.min_length_5,
+      maximum: Settings.user.namee.max_length_50
```

Comment:
nếu em truyền `user.namee` vào thì khó tái sử dụng được code
-> đổi lại thành 
```suggestion
      maximum: Settings.validations.digit.length_50
```